gum/backend-windows/gumprocess-windows.c

```diff
@@ -305,13 +305,77 @@ _gum_process_enumerate_ranges (GumPageProtection prot,
   }
 }
 
+static gboolean
+gum_process_enumerate_heap_ranges (HANDLE heap,
+                                   GumFoundMallocRangeFunc func,
+                                   gpointer user_data)
```

Comment:
This function is lower level than the function below that's using it, so it should be right below it instead. (And forward-declaration should be at the top of the file, ordered chronologically relative to the other forward declarations.)